From: uazo <uazo@users.noreply.github.com>
Date: Wed, 10 Sep 2025 07:51:21 +0000
Subject: Set caret blink interval to default

Disable the use of the system setting for the caret blink interval

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 third_party/blink/renderer/core/exported/web_view_impl.cc | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/third_party/blink/renderer/core/exported/web_view_impl.cc b/third_party/blink/renderer/core/exported/web_view_impl.cc
--- a/third_party/blink/renderer/core/exported/web_view_impl.cc
+++ b/third_party/blink/renderer/core/exported/web_view_impl.cc
@@ -3606,9 +3606,7 @@ void WebViewImpl::UpdateRendererPreferences(
   UpdateFontRenderingFromRendererPrefs();
 
   blink::SetCaretBlinkInterval(
-      renderer_preferences_.caret_blink_interval.has_value()
-          ? renderer_preferences_.caret_blink_interval.value()
-          : base::Milliseconds(
+          base::Milliseconds(
                 mojom::blink::kDefaultCaretBlinkIntervalInMilliseconds));
 
 #if defined(USE_AURA)
--
